docs/windows.md

```diff
@@ -59,7 +59,54 @@ and daemons are pre-installed on the Windows Nodes in the demo.
     See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes)
     section.
 
-### Installation
+### Installation as a Service (Containerd based runtimes)
+
+First install Antrea (v0.13.0+ is required for Containerd).
+
+```bash
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.13.0/antrea.yml
+```
+
+Then, you can run the following commands. This will install Antrea as a Windows service. Please ensure [nssm](https://nssm.cc/)
+is on your machine, which is a handy tool to manage service on Windows. NOTE: `<KubernetesVersion>`, `<KubeconfigPath>`
+and `<KubeletKubeconfigPath>` should be filled by you. E.g.
+```powershell
+$KubernetesVersion="v1.20.4"
+$KubeConfig="C:/Users/Administrator/.kube/config" # admin kubeconfig
+$KubeletKubeConfig="C:/etc/kubernetes/kubelet.conf"
+```
+
+```powershell
+$TAG="v0.13.0"
+$KubernetesVersion="<KubernetesVersion>"
+$KubeConfig="<KubeconfigPath>"
+$KubeletKubeConfig="<KubeletKubeconfigPath>"
+$KubernetesHome="c:/k"
+$AntreaHome="c:/k/antrea"
+
+curl.exe -LO "https://raw.githubusercontent.com/vmware-tanzu/antrea/${TAG}/hack/windows/Helper.psm1"
+
+Import-Module ./helper.psm1
+Install-AntreaAgent -KubernetesVersion "$KubernetesVersion" -KubernetesHome "$KubernetesHome" -KubeConfig "$KubeConfig" -AntreaVersion "$TAG" -AntreaHome "$AntreaHome"
+New-KubeProxyServiceInterface
+
+mkdir "${KubernetesHome}/antrea/logs"
+nssm install kube-proxy "${KubernetesHome}/kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=$KubeletKubeConfig --log-dir=c:/var/log/kube-proxy --logtostderr=false --alsologtostderr"
```

Comment:
What about use config file like `KubeProxyKubeConfig` instead of `KubeletKubeConfig`. It means let user specify a kubeconfig file used by kube-proxy. The kubelet configfile contains too many permissions.

For example, user could generate the `KubeProxyKubeConfig` use the `kube-proxy` service account.

Reply:
@lzhecheng any thoughts on that, is there any upstream reference on running kube-proxy as a service?

Reply:
@antoninbas this doc uses `sc.exe` but `nssm` is similar here.
https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#troubleshooting

Reply:
`KubeletKubeConfig` is changed to `KubeProxyKubeconfigPath`.